### PR TITLE
Added Docker aliases

### PR DESCRIPTION
**Pull request by BrentWilkins**
I copied the Docker aliases from [ohmyzsh](https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/docker/docker.plugin.zsh)'s Docker plugin.
@akinomyoga

### CODE REVIEW

File: aliases/README.md [hunk 1]
```diff
@@ -76,4 +76,42 @@ OMB_ALIAS_PACKAGE_MANAGER_SUDO=
 | `tinit`  | `terraform init`             |
 | `tplan`  | `terraform plan`             |
 | `tapply` | `terraform apply`            |
-| `tfmt`   | `terraform fmt`              | 
+| `tfmt`   | `terraform fmt`              |
+
+## alias:Docker
```
Comment (akinomyoga) [inline aliases/README.md:81]: Can you put this section before `alias:package-manager`? Alphabetical order is useful for finding the corresponding section.

**Review by akinomyoga — CHANGES_REQUESTED** [review]
Thank you for your contribution.

File: aliases/README.md (hunk 1 above)
Comment (BrentWilkins) [reply aliases/README.md]: Thanks, I appreciate sorting too. I also added backticks (\<code>alias_name\</code>) around the alias names too. ohmyzsh used a different format.

File: aliases/README.md [hunk 2]
```diff
@@ -47,6 +47,44 @@ To activate it, add `debian` to `plugins(...)` in your `.bashrc` file:
 | `upgrb` | `sudo update-grub`         |
 | `uirfs` | `sudo update-initramfs -u` |
 
+## alias:Docker
+
+|  Alias  |  Command                      |  Description                                                                               |
+| :------ | :---------------------------- | :----------------------------------------------------------------------------------------- |
+| `dbl`     | `docker build`                | Build an image from a Dockerfile                                                         |
```
Comment (akinomyoga) [inline aliases/README.md:54]:
```suggestion
|  Alias    |  Command                      |  Description                                                                             |
| :-------- | :---------------------------- | :--------------------------------------------------------------------------------------- |
| `dbl`     | `docker build`                | Build an image from a Dockerfile                                                         |
```

**Review by akinomyoga — APPROVED** [review]

**Pull request merged**